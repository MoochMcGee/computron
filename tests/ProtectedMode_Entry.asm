[bits 16]

mov eax, 1
mov cr0, eax

db 0xf1
