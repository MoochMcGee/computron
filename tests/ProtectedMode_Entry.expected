1000:00000000 66 EAX=00000000 EBX=00000000 ECX=00000000 EDX=00000000 ESP=00000000 EBP=00000000 ESI=00000000 EDI=00000000 CR0=00000000 A20=0 DS=0000 ES=0000 SS=0000 FS=0000 GS=0000 C=0 P=0 A=0 Z=0 S=0 I=1 D=0 O=0 A16 O16 X16
1000:00000001 B8 EAX=00000000 EBX=00000000 ECX=00000000 EDX=00000000 ESP=00000000 EBP=00000000 ESI=00000000 EDI=00000000 CR0=00000000 A20=0 DS=0000 ES=0000 SS=0000 FS=0000 GS=0000 C=0 P=0 A=0 Z=0 S=0 I=1 D=0 O=0 A16 O32 X16
1000:00000006 0F EAX=00000001 EBX=00000000 ECX=00000000 EDX=00000000 ESP=00000000 EBP=00000000 ESI=00000000 EDI=00000000 CR0=00000000 A20=0 DS=0000 ES=0000 SS=0000 FS=0000 GS=0000 C=0 P=0 A=0 Z=0 S=0 I=1 D=0 O=0 A16 O16 X16
1000:00000009 F1 EAX=00000001 EBX=00000000 ECX=00000000 EDX=00000000 ESP=00000000 EBP=00000000 ESI=00000000 EDI=00000000 CR0=00000001 A20=0 DS=0000 ES=0000 SS=0000 FS=0000 GS=0000 C=0 P=0 A=0 Z=0 S=0 I=1 D=0 O=0 A16 O16 X16
